CactbotOverlay/CactbotOverlay.cs

```diff
@@ -301,7 +301,7 @@ public CactbotOverlay(CactbotOverlayConfig config)
           }
         }
 
-        var web = new System.Net.WebClient();
+				var web = new System.Net.WebClient();
```

Comment:
Sorry, but can you convert all these tabs to spaces, here and elsewhere?

I really need to have a style checker commit hook, but I haven't gotten there yet.

Reply:
sorry, yes, work uses tabs so I have VS set to use tabs and I forget about it.  I'll fix this and send it.

Reply:
I've changed my VS settings for this and did an auto-format to fix anywhere that I still had tabs, so my pull in should be fine now.